tests/hap/test_svm_wfc3ir.py

```diff
@@ -0,0 +1,213 @@
+""" This module tests full pipeline SVM processing as a demonstration template.
+
+"""
+import datetime
+import os
+import pytest
+import numpy as np
+
+from drizzlepac.haputils import astroquery_utils as aqutils
+from drizzlepac import runsinglehap
+from astropy.io import fits, ascii
+from pathlib import Path
+
+"""
+    test_svm_demo.py
+
+    This test file can be executed in the following manner:
+        $ pytest -s --basetemp=/internal/hladata/yourUniqueDirectoryHere test_svm.py >& test_svm.log &
+        $ tail -f test_svm.log
+      * Note: When running this test, the `--basetemp` directory should be set to a unique
+        existing directory to avoid deleting previous test output.
+      * The POLLER_FILE exists in the tests/hap directory.
+
+    *** The --basetemp does not seem to be working at this time!!!
```

Comment:
Suggest to update the commentary to be specific to this test and remove the --basetemp warning which is obsolete due to the tmpdir addition.

Reply:
Fixed in 2893c91.